Dkfz config (#441)
nf-core/configs: docs/dkfz.md

# nf-core/configs: DKFZ configuration

This configuration specifies hardware and infrastructure access at the Deutsches Krebsforschungszentrum (DKFZ) HPC cluster in Heidelberg / Germany.

> To use, run the pipeline with `-profile dkfz`. This will download and launch the dkfz.config which has been pre-configured with a setup suitable for the cluster. Using this profile, a Docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.

Reviewer: ```suggestion
To use, run the pipeline with `-profile dkfz`. This will download and launch the dkfz.config which has been pre-configured with a setup suitable for the cluster. Using this profile, either Singularity containers are pulled from public repositories or a Docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
```

Author: config file is changed now, and it is more suitable for usage of DKFZ in general.

## DKFZ configuration

> This is a configuration file has been used in DKFZ HPC cluster. 

Reviewer: ```suggestion
This configuration specifies hardware and infrastructure access at the Deutsches Krebsforschungszentrum (DKFZ) HPC cluster in Heidelberg / Germany.
```

> To use, run the pipeline with -profile dkfz. This will download and launch the dkfz.config which has been pre-configured with a setup suitable for the automatically created cluster. Using this profile, a Docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.

Reviewer: Is that cluster really created automatically for you or is it just available?

Author: it is available.

Reviewer: Then I would reformulate the above - it sounds like its dynamically created once you run a job which is not the case here ;-)

Before running the pipeline you will need to load Nextflow using the environment module system. Please check the main README of the pipeline to make sure that the version of Nextflow is compatible with that required to run the pipeline. You can do this by issuing the commands below:

  ```console
	module load Nextflow/21.04.0
  ```

> In order to use singularity images from a directory this line must be added to the config env:

Reviewer: ```suggestion
In order to use singularity images from a directory, this line must be added to the config `env`:
```
